lib/src/model.dart

```diff
@@ -3760,10 +3778,7 @@ class Operator extends Method {
 
 // TODO(jcollins-g): Break [Package] out into a real single-package only
 // class, and a container class for a set of packages.
-class Package extends Canonicalization with Nameable, Warnable {
-  // Library objects serving as entry points for documentation.
-  final List<Library> _libraries = [];
-
+class PackageGraph extends Canonicalization with Nameable, Warnable {
```

Comment:
I'm not sure I would call it a PackageGraph myself (`PackageGroup`? `Project`?), but whatever you're comfortable with.

Reply:
Eventually, this object will represent the collection of Packages and their cross-links via dependencies being documented.  I admit that `PackageGraph` is more ambition than reality at this point.